### ISSUE TITLE
prefer-toBeUndefined produces AssertionError on found error

### ISSUE THREAD

**Issue by pascal-puetz** [tlvince/eslint-plugin-jasmine#355]
**Describe the bug**

The `jasmine/prefer-toBeUndefined` rule produces an Assertion error when any spec file is found that contains a hit (e.g. `expect(undefined).toBe(undefined)` with `{"jasmine/prefer-toBeUndefined": "error"}` config).

The error only occurs on versions 4.x.x and did not occur on 3.0.0.

**To Reproduce**

Set the `jasmine/prefer-toBeUndefined` config to `"error"` and produce an error in any spec file. Afterwards, lint that file. This produces the following error and crashes eslint:

```
Oops! Something went wrong! :(

ESLint: 8.22.0

AssertionError [ERR_ASSERTION]: Fix has invalid range: {
  "range": [
    null,
    null
  ],
  "text": "toBeUndefined("
}
```

**Expected behaviour**

Eslint should report the error as usual and should not crash.

**Context**

- Node.js version: 16.4.0
- eslint-plugin-jasmine version: 4.1.3
- eslint version: 8.22.0
- Operating system: MacOS Monterey 12.2.1
